src/main/java/frc/robot/Robot.java

```diff
@@ -59,22 +59,20 @@ public class Robot extends TimedRobot  {
    */
   @Override
   public void robotInit() {
-    // Instantiate our RobotContainer.  This will perform all our button bindings, and use the subsystems needed
-    // for the specific robot
-    // LimelightHelpers.setStreamMode_PiPSecondary("limelight");
-
-    // Shuffleboard.getTab("GameTab").addCamera("Vision", "limelight", "http://limelight.local:5800").withSize(4,3).withPosition(5, 0);
-
     // String logfolder = "/home/lvuser";
     // Logger.addDataReceiver(new WPILOGWriter(logfolder));
     // Logger.addDataReceiver(new NT4Publisher());
     // Logger.start();
 
-    //SignalLogger.enableAutoLogging(true);
-    //SignalLogger.start();
+    SignalLogger.setPath("/home/lvuser/log/");
```

Comment:
Do we want these in the code during game time?
Do we want this for all containers, even the test ones?